index.html

```diff
@@ -321,6 +321,15 @@ <h5 style="text-align: center; color:aliceblue; font-size: large; ">
 		</h5>
 	</section>
 
+	
+	<section class="bg-white">
+		<div class="container" style="text-align: center;">
+			<h4 style="font-size: 2rem; line-height: 1.5; margin: 0; letter-spacing: 3px;">Meshery is in the CNCF Landscape</h4>
```

Comment:
It will be a good practice if you move the in-line styling to its respective styling file.

Reply:
Done! thank you